code/modules/mob/mob.dm

```diff
@@ -362,7 +362,7 @@
 
 		add_logs(src, M, "grabbed", addition="passive grab")
 		if(!supress_message)
```

Comment:
This isn't relevant, but I thought it might be funny if this was called `suplex_message` instead.

Reply:
:joy: